ehrapy/tools/_scanpy_tl_api.py

```diff
@@ -774,8 +765,9 @@ def rank_features_groups(
     pts: bool = False,
     key_added: Optional[str] = "rank_features_groups",
     copy: bool = False,
-    method: _rank_features_groups_method = None,
-    corr_method: _corr_method = "benjamini-hochberg",
+    method: _datatypes._rank_features_groups_method = None,
+    categorical_method: _datatypes._rank_features_groups_cat_method = "g-test",
+    corr_method: _datatypes._corr_method = "benjamini-hochberg",
```

Comment:
I totally agree with this confusion. I would rename the argument. Otherwise, users have to read docs or code to understand what it does. I would actually rename it in scanpy as well